File: ceshi.py
# -*- coding:utf-8 -*-

import requests
import pprint
import json
import re
import sys

# try:
#     ndoc_id =sys.argv[2]
#     url = 'http://www.looklaw.cn/docdemo/structor?anyou_code=00100500'+sys.argv[1]+'&id_no='+ndoc_id
# except:
#     ndoc_id = ""
#     url = 'http://www.looklaw.cn/docdemo/structor?anyou_code=00100500' + sys.argv[1]

# try:
#     ndoc_id =sys.argv[1]
#     url = 'http://www.looklaw.cn/docdemo/structor?anyou_code=001004009&id_no='+sys.argv[1]
# except:
#     url = 'http://www.looklaw.cn/docdemo/structor?anyou_code=001004009'


#url = "http://www.looklaw.cn/docdemo/structor?anyou_code=001006001020"
# url = "http://www.looklaw.cn/docdemo/structor?anyou_code=001005002&id_no=cfd66deb6f085d8430526394c6fc4683"
#url = "http://www.looklaw.cn/docdemo/structor?anyou_code=001004003"
# url = "http://www.looklaw.cn/docdemo/structor?anyou_code=001004009"

doc_id =sys.argv[1]
url = "http://www.looklaw.cn/docdemo/structor?id_no="+sys.argv[1]
# 22af7e0a1fe01d49b67d74ab5686ee3c
pp = pprint.PrettyPrinter(indent=2)
response = requests.get(url)
res = response.json()
resp = res[0]
case_num = resp['case_num']
id_no = resp['id_no']
doc_id = resp['id_no' ]
case_type = resp['case_type']
anyou_code = resp['anyou_code']
anyou_name = resp['anyou_name']
fayuan_name = resp['fayuan_name']
origin_content = resp['origin_content']
fayuan_renwei = resp['fayuan_renwei']
print("")
print("")
print("")
print(fayuan_renwei)
print("")
print("")
print("")
print(id_no)
print("")
print("")
print("")
print(origin_content)

case_url = 'http://127.0.0.1:3000/index.php'
data = {
    'case_num': case_num,
    'doc_id': doc_id,
    'case_type': case_type,
    'anyou_code': anyou_code,
    'anyou_name': anyou_name,
    'fayuan_name': fayuan_name,
    'origin_content': origin_content
}
html = requests.post(case_url, data=data)
print(html.text)
pp.pprint(html.json())

File: ceshi.rb
#!/usr/bin/ruby
# coding: utf-8
require 'pry'
require 'pp'
require 'json'
require 'net/http'
require 'uri'

# http://www.looklaw.cn/doc/e12848ce5d9f197ddc327819ddeeadf4
id_no = ARGV[0]
url = "http://www.looklaw.cn/docdemo/structor?id_no="+id_no
# binding.pry
uri = URI.parse url
result =  Net::HTTP.get uri
res = JSON.parse result

resp = res[0]
case_num = resp['case_num']
id_no = resp['id_no']
doc_id = resp['doc_id' ]
case_type = resp['case_type']
anyou_code = resp['anyou_code']
anyou_name = resp['anyou_name']
fayuan_name = resp['fayuan_name']
origin_content = resp['origin_content']
fayuan_renwei = resp['fayuan_renwei']
doc_type = resp['doc_type']
judge_level = resp['judge_level']

pp resp

# pp origin_content
puts '*'*150
# 调用本地接口测试
url = URI.parse('http://localhost:3000/looklaw')

if case_type == '执行案件'
  judge_level = ''
end

data = {
    'case_type': case_type.sub('案件',''),
    'judge_level': judge_level,
    'doc_type': doc_type.sub('书',''),
    'content': origin_content
}
begin
  response = Net::HTTP.post_form(url,data)
  # binding.pry
  res = JSON.parse response.body
  pp res
rescue
  puts id_no
end
